xray/commands/audit/scan.go

```diff
@@ -246,19 +245,22 @@ func (scanCmd *ScanCommand) performScanTasks(fileConsumer parallel.Runner, index
 				violations = append(violations, res.Violations...)
 				vulnerabilities = append(vulnerabilities, res.Vulnerabilities...)
 			}
-			if len(res.Violations) > 0 {
-				// A violation was found, the scan failed.
+			if len(res.Violations) > 0 || len(res.Vulnerabilities) > 0 {
+				// A violation or vulnerability was found, the scan failed.
 				passScan = false
 			}
 		}
 	}
+	fmt.Println("The full scan results are available here: " + tempDirPath)
 	if len(violations) > 0 {
 		err = xrutils.PrintViolationsTable(violations)
 	}
 	if len(vulnerabilities) > 0 {
 		xrutils.PrintVulnerabilitiesTable(vulnerabilities)
 	}
-	// No violations found, return scan OK.
+	if passScan {
+		fmt.Println("The scan completed successfully.")
```

Comment:
fmt.Println("The scan completed successfully.")
-->
log.Info("Scan completed successfully.")